core/src/main/java/com/scalar/db/transaction/consensuscommit/TwoPhaseConsensusCommitManager.java

```diff
@@ -278,14 +267,8 @@ public void mutate(List<? extends Mutation> mutations)
   private <R> R executeTransaction(
       ThrowableFunction<TwoPhaseCommitTransaction, R, TransactionException> throwableFunction)
       throws CrudException, UnknownTransactionStatusException {
-    TwoPhaseCommitTransaction transaction;
-    try {
-      transaction = beginInternal();
-    } catch (TransactionNotFoundException e) {
-      throw new CrudConflictException(e.getMessage(), e, e.getTransactionId().orElse(null));
-    } catch (TransactionException e) {
-      throw new CrudException(e.getMessage(), e, e.getTransactionId().orElse(null));
-    }
+    String txId = UUID.randomUUID().toString();
+    TwoPhaseCommitTransaction transaction = begin(txId);
```

Comment:
Same as above

Reply:
Fixed in 41b597a0ce93ed0b3c9b9d3d3bd475eabde82c22. Thanks!